src/view/abstract-element.ts

```diff
@@ -0,0 +1,25 @@
+import { createElement } from '../render';
+
+export interface TemplateElement {
+  template: string;
+  element: Element;
+  removeElement: () => void;
+}
+
+export abstract class AbstractElement {
+  _element: Element | null = null;
+
+  abstract get template(): string;
+
+  get element(): Element {
+    if (!this._element) {
+      this._element = createElement(this.template);
+    }
```

Comment:
Мне кажется так короче
```suggestion
  get element(): E {
    this.#element ??= createElement<E>(this.template);
    return this.#element;
  }
```